wiki/osu!_File_Formats/Osu_(file_format)/en.md

```diff
@@ -142,15 +142,15 @@ OverallDifficulty:4
 ApproachRate:4.8
 ```
 
-*HPDrainRate* specifies how fast the health bar decreases. At the time of this writing, the near future definition of this property remains unsettled.
+*HPDrainRate* specifies how fast the health decreases. However, the definition of this property may be changed in the near future.
```

Comment:
Shouldn't this be under a section called `### HP Drain Rate` and the abbreviation for HPDrainRate be *HP*?